### PR TITLE
DataMiner Credits

### PR DESCRIPTION
**Pull request by JordyGit**

### CODE REVIEW

File: user-guide/Cloud_Platform/Credits/DataMiner_Credits.md
Comment (AlexanderVK-SLC) [inline user-guide/Cloud_Platform/Credits/DataMiner_Credits.md:9]: Sounds weird to say 'for a period of 2 weeks', almost as if you get the credit back after this time.

Comment (JordyGit) [reply user-guide/Cloud_Platform/Credits/DataMiner_Credits.md:9]: Later it might be renewed for another 2 weeks for another credit, not sure how to say it better?

Comment (AlexanderVK-SLC) [reply user-guide/Cloud_Platform/Credits/DataMiner_Credits.md:9]: 'The DataMiner as a Service (DaaS) system will deduct 1 credit every 2 weeks as long as it remains active'

Comment (MariekeGO) [reply user-guide/Cloud_Platform/Credits/DataMiner_Credits.md:9]: How about this?

```suggestion
Credits will be deducted automatically when you consume usage-based services. For example, if you deploy a DataMiner as a Service (DaaS) system for a period of 2 weeks, one credit will be deducted. <!-- link to DaaS docs -->>
```

File: user-guide/Cloud_Platform/toc.yml
Comment (MariekeGO) [inline user-guide/Cloud_Platform/toc.yml:1]: I'm not sure about this location. I'm thinking of maybe moving it up so it's in between "Connecting your DataMiner System to dataminer.services" and "Project Collaboration", or moving it below the "About dataminer.services" node (above "Cloud data storage policies"), as it seems a good fit there. What do you think?

**Comment by baptistepa** [comment]
Wait for Bert VDB to give the go-ahead to merge this.

**Review by baptistepa — CHANGES_REQUESTED** [review]
Waiting for @BertVandenberghe to give the go ahead.

**Comment by baptistepa** [comment]
Feedback for some rework:

- it will cost 3 credits to host a DataMiner for 1 week
- change the title of this docs to DataMiner Express Edition instead of focusing on credits
- explain what the edition is (limited DataMiner edition)
- move the docs under dataminer.services instead of having a separate submenu
- link to credit docs from sales: https://docs.dataminer.services/dataminer-overview/Pricing/Pricing_Usage_based_service.html

**Comment by MariekeGO** [comment]
> Feedback for some rework:
> 
> * it will cost 3 credits to host a DataMiner for 1 week
> * change the title of this docs to DataMiner Express Edition instead of focusing on credits
> * explain what the edition is (limited DataMiner edition)
> * move the docs under dataminer.services instead of having a separate submenu
> * link to credit docs from sales: https://docs.dataminer.services/dataminer-overview/Pricing/Pricing_Usage_based_service.html

I'm wondering, if we're going to change the focus of this page entirely to explaining what DataMiner Express Edition is, wouldn't it fit better under DataMiner Overview with the other Sales content? It seems to be going outside the scope of the User Guide then (though we can of course link to the Sales page in a note where appropriate).

**Comment by MariekeGO** [comment]
@baptistepa
I've reworked the page based on the feedback above; however, some things still need to be cleared up:

- Where do we add this? This looks like something for the [DataMiner Overview](https://docs.dataminer.services/dataminer-overview/Overview_index.html) section, but it might be better to check with Sales before we add this there.
- What should happen with this section: https://docs.dataminer.services/dataminer-overview/Pricing/Pricing_Commercial_Models.html#free-dataminer-system? It looks like this will at least partly become outdated based on this new content.
- Previously, this PR mentioned that every organization receives 1 DataMiner Credit to try out a DaaS System. Is it still the plan for them receive free credits? If yes, how many?

**Comment by JordyGit** [comment]
> @baptistepa I've reworked the page based on the feedback above; however, some things still need to be cleared up:
> 
> * Where do we add this? This looks like something for the [DataMiner Overview](https://docs.dataminer.services/dataminer-overview/Overview_index.html) section, but it might be better to check with Sales before we add this there.
> * What should happen with this section: https://docs.dataminer.services/dataminer-overview/Pricing/Pricing_Commercial_Models.html#free-dataminer-system? It looks like this will at least partly become outdated based on this new content.
> * Previously, this PR mentioned that every organization receives 1 DataMiner Credit to try out a DaaS System. Is it still the plan for them receive free credits? If yes, how many?

3 Credits will be the new free default value

**Review by baptistepa — APPROVED** [review]

**Comment by MariekeGO** [comment]
> @MariekeGO Can we add this under the section from Free DataMiner and rework that section? My idea was to have this as follows
> 
> # Express DataMiner
> general info
> 
> ## On-prem
> specific info about on prem deployments
> 
> ## Cloud
> specific information about cloud deployments like the credits

I've made the necessary changes based on your feedback. Could you check if this is OK for you now?

**Comment by baptistepa** [comment]
Thanks I will double check with Bert and hopefully we can merge this today :)

**Comment by baptistepa** [comment]
> Thanks I will double check with Bert and hopefully we can merge this today :)

Bert confirmed this is okay for him.

**Pull request merged**
